content/en/metrics/open_telemetry/_index.md

```diff
@@ -0,0 +1,158 @@
+
+## Setup
+
+### OTLP ingestion in the Datadog Agent {#otlp-ingest}
+**OTel SDKs/Libraries** -> **Datadog Agent** -> **Datadog**
+
+#### Use case
+- You want a full range of Datadog capabilities (Datadog Agent capabilities, infrastructure monitoring, container monitoring, integrations).
+- You are collecting telemetry data from OTel SDKs and the Datadog Agent. You do not plan on collecting data from other sources.
+- You are routing telemetry data to Datadog. You do not plan on routing data to other vendors.
+
+#### Configuration
+
+1. [Instrument your system with OpenTelemetry][1].
+2. [Enable OTLP ingestion][2] on the Datadog Agent.
+3. [Send OTLP telemetry][3] from your system to Datadog.
+
+### OTLP Exporter {#otlp-exporter}
+**OTel SDKs/Libraries** -> **OTel Collector** with the OTLP Exporter -> **Datadog Agent** -> **Datadog**
+
+#### Use case
+- You want a full range of Datadog capabilities (Datadog Agent capabilities, infrastructure monitoring, container monitoring, integrations).
+- You are collecting telemetry data from sources besides OTel SDKs and the Datadog Agent. For example: Prometheus metrics.
+- You are routing telemetry data to multiple vendors.
+- You are deploying the OTel Collector and Datadog Agent on the same host.
+
+#### Configuration
+
+1. [Instrument your system with OpenTelemetry][1].
+2. [Enable OTLP ingestion through gRPC][2] on the Datadog Agent.
+3. Add an OTLP exporter to your [OTel configuration YAML file][4]. Point the exporter to the Datadog Agent endpoint. For example, if your Datadog Agent is listening on port 4317 and you are running the OTel Collector on the same host, you may define the exporter as:
+   ```yaml
+   receivers:
+     otlp:
+         protocols:
+             grpc:
+               # Set to different port from Datadog Agent OTLP ingest 
+               # Point your instrumented application to port '5317' if using gRPC.
+               endpoint: "0.0.0.0:5317" 
+             http:
+               # Set to different port from Datadog Agent OTLP ingest 
+               # Point your instrumented application to port '5318' if using HTTP.
+               endpoint: "0.0.0.0:5318"
+
+     hostmetrics:
+       scrapers:
+         load:
+         cpu:
+         disk:
+         filesystem:
+         memory:
+         network:
+         paging:
+         process:
+
+   processors:
+     batch:
+       timeout: 10s
+
+   exporters:
+     otlp:
+       endpoint:
+         key: "0.0.0.0:4317"
+         tls:
+           insecure: true
+        
+   service:
+     pipelines:
+       metrics:
+         receivers: [hostmetrics]
+         processors: [batch]
+         exporters: [otlp]
+   ```
+   If you are using a containerized environment, ensure the `endpoint` setting is configured to use the approprate hostname for the Datadog Agent
+
+### Datadog Exporter {#datadog-exporter}
+**OTel SDKs/Libraries** -> **OTel Collector** with the Datadog Exporter -> **Datadog**
```

Comment:
Same here. Can we add diagram instead? Here's what the blog post has: https://a.cl.ly/mXuxeYbm Should we directly use that picture?